packages/core/src/child-proxy/ChildProcessProxy.ts

```diff
@@ -27,6 +27,7 @@ const BROKEN_PIPE_ERROR_CODE = 'EPIPE';
 const IPC_CHANNEL_CLOSED_ERROR_CODE = 'ERR_IPC_CHANNEL_CLOSED';
 const TIMEOUT_FOR_DISPOSE = 2000;
 
+let n = 0;
```

Comment:
What is this doing here? `n` is never updated.

Reply:
`n` is updated on 56 and used on 55. Basically, I use it to enumerate the child process output files to check how many of them are missing. Another naming scheme is fine with me.

Reply:
Should we not reuse this `n`? For example, with `--concurrency 4`, we might want to use `1`, `2`, `3`, and `4`, but never `5`. Whenever a child process crashes, it can reuse the number.

Reply:
No, that'll result in overwritten `.cpuprofile` files.

`n` here is not a good solution by any means, but it is a human-readable approach to unique file names. Doing it sequenced like this allows a human to see how many processes crashed, as crashed processes have no `.cpuprofile` file.

For example:

![image](https://user-images.githubusercontent.com/22448210/95969051-8940fa00-0e0e-11eb-9191-cad27371259a.png)

We can see that 1, 3, 6-10, 12-14, and 17 are missing here. These processes must have crashed for some reason.

This insight is the only reason why we need `n`.